alacritty.yml

```diff
@@ -313,7 +313,9 @@
 #background_opacity: 1.0
 
 #selection:
-  #semantic_escape_chars: ",│`|:\"' ()[]{}<>\t"
+  # This characters for the boundaries when quickly selecting words
+  # Remove ':' if you frequently select URLs so the whole URL matches.
```

Comment:
I don't think we should recommend removing `:`. If people don't like these, they can decide on their own what they want to remove.

Reply:
OK. I added it thinking selecting URLs would be a common use-case for the feature, as the various other approaches for selecting URLs with the keyboard have not been implemented.

Reply:
I've removed this line in my updated commit.